helios/stats_views.py

```diff
@@ -40,13 +40,15 @@ def elections(request):
 
   page = int(request.GET.get('page', 1))
   limit = int(request.GET.get('limit', 25))
+  q = request.GET.get('q','')
```

Comment:
yeah, that's what I get for commenting on PRs too late. Looks good on that front. The one area I'm still worried about is using `{{q}}` in the `href`, could you `urlencode` those? Thanks!